components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -561,7 +561,7 @@ bool JsonRpcService::SetNetwork(const std::string& chain_id,
   SetCurrentChainId(prefs_, coin, origin, chain_id);
 
   if (!silent) {
-    FireNetworkChanged(coin, origin);
+    FireNetworkChanged(coin, GetChainId(coin, origin), origin);
```

Comment:
just chain_id rather than calling GetChainId?

Reply:
fixed in d3e6a82b8a6f32905c685c16ba433b87e5126973